src/components/Header/components/PeopleSearch/index.js

```diff
@@ -349,7 +370,13 @@ export default class GordonPeopleSearch extends Component {
                       }),
                     )}
                   </Paper>
-                ) : null}
+                ) : isOpen && this.state.suggestions.length === 0 &&
```

Comment:
Is there a way to avoid having 2 copies of this code? (Here and line 333)

Reply:
I'm not sure there's a simple way to do this. because I need lines 377 and 379 to hold line 378 and when I put that into a function it doesn't work as intended. If you're talking about the if else statements, I'm also not sure if it would be more efficient to turn it into a function because I think I would need to pass isOpen to it.

Reply:
I was thinking of the whole if/else, with almost-identical <Downshift> inside each.   I'm concerned with DRY code (don't repeeat yourself) for correctness.  I don't think efficiency is different enough to worry about.  Since it's a larger problem than just your change, let's submit this.  

If you have time afterwards, it would be nice to refactor (if possible) so there's one copy with the "online" parameter used internally.